staff/groups/aleti/project-app/src/logic/UserApi.js

```diff
@@ -0,0 +1,124 @@
+const UserApi = {
```

Comment:
IDEM before -> lowerCamelCase

Reply:
BTW userApi does not seem to be hiding the detail of being an API, instead, may "usersLogic" fit better (the implementation details should not be exposed... now it can be an API, but tomorrow in could be another thing, so just "usersLogic" works fine)

Reply:
+1